include/crow/json.h

```diff
@@ -1224,15 +1224,40 @@ namespace crow
         class wvalue : public returnable
         {
             friend class crow::mustache::template_t;
+
+        public:
+            using object_type =
+#ifdef CROW_JSON_USE_MAP
+            std::map<std::string, wvalue>;
+#else
+            std::unordered_map<std::string, wvalue>;
+#endif
+
         public:
             type t() const { return t_; }
         private:
             type t_{type::Null}; ///< The type of the value.
             num_type nt{num_type::Null}; ///< The specific type of the number if \ref t_ is a number.
-            union {
+            union number {
               double d;
               int64_t si;
-              uint64_t ui {};
+              uint64_t ui;
+
+            public:
+              constexpr number() noexcept : ui() {} /* default constructor initializes unsigned integer. */
+
+              constexpr number(std::uint8_t value) noexcept : ui(static_cast<std::uint64_t>(value)) {}
```

Comment:
Why these casts?

Reply:
I always explicilty cast if I'm are sure that this is the intended operation even if there is no warnings here I think this good practice to do it as soon as end-user will not notice any issue there.